seed/ts-sdk/bytes/src/api/resources/service/client/Client.ts

```diff
@@ -78,4 +78,53 @@ export class Service {
                 });
         }
     }
+
+    /**
+     * @param {Service.RequestOptions} requestOptions - Request-specific configuration.
+     */
+    public async download(requestOptions?: Service.RequestOptions): Promise<ArrayBuffer> {
```

Comment:
Critical change

Reply:
In that case Response is the best option, it can be streamed, turned into a buffer, blob, etc